compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -166,10 +166,10 @@ class ComposeScene internal constructor(
         }
     }
```

Comment:
When I make run1's popup focusable, I can't close it:
```
Subject: [PATCH] 1
---
Index: compose/desktop/desktop/samples/src/jvmMain/kotlin/androidx/compose/desktop/examples/example1/Main.jvm.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/compose/desktop/desktop/samples/src/jvmMain/kotlin/androidx/compose/desktop/examples/example1/Main.jvm.kt b/compose/desktop/desktop/samples/src/jvmMain/kotlin/androidx/compose/desktop/examples/example1/Main.jvm.kt
--- a/compose/desktop/desktop/samples/src/jvmMain/kotlin/androidx/compose/desktop/examples/example1/Main.jvm.kt	(revision 63c9f6248f465d2109858bfc6c7fb55a85d26aed)
+++ b/compose/desktop/desktop/samples/src/jvmMain/kotlin/androidx/compose/desktop/examples/example1/Main.jvm.kt	(date 1689348611751)
@@ -512,6 +512,7 @@
         }
         DropdownMenu(
             expanded = dropDownMenuExpanded.value,
+            focusable = false,
             onDismissRequest = {
                 dropDownMenuExpanded.value = false
                 println("OnDismissRequest")

```


https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/7df20fa0-0943-4143-b698-16215a53e4ab

Reply:
It wasn't changed in this PR

Reply:
This is expected and worked this way before